cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -664,6 +664,26 @@ def _validate(self, storage_type: str, max_number: int, storage_count: int):
             )
 
 
+class ManagedFsxStorageMultipleSubnetsValidator(Validator):
+    """
+    Managed FSx Storage Vs Multiple Subnets validator.
+
+    Validate if managed storage of type FSx is set when using multiple subnets in queues configuration.
+    """
+
+    def _validate(self, queues, new_storage_count):
+        if any(len(queue.networking.subnet_ids) > 1 for queue in queues) and new_storage_count.get("fsx") > 0:
+            self._add_failure(
+                "Multiple subnets configuration does not support FSx 'managed' storage.{0} Please make sure to provide "
+                "an existing shared storage, properly configured to work across the target subnets.".format(
+                    " Found {0} 'managed' FSx storage.".format(new_storage_count.get("fsx"))
+                    if new_storage_count.get("fsx") > 0
```

Comment:
This block is already under the `new_storage_count.get("fsx") > 0` condition.

Reply:
fixed